ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue

```diff
@@ -0,0 +1,217 @@
+                        partnership. The home must be registered in the name of the sole proprietor or partner (person
+                        or business).
+                      </v-tooltip>
+                    </v-col>
+                  </v-row>
+                </v-list-item-subtitle>
+              </v-list-item-content>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+        <v-list v-else-if="!isSearchResultSelected && showDropdown && !searching && autoCompleteResults.length === 0">
+          <v-list-item class="auto-complete-item">
+            <v-list-item-content class="px-2 pt-1 pb-0">
+              <v-list-item-subtitle>
+                <v-row class="auto-complete-row">
+                  <v-col id="no-party-matches">
+                    <p>
+                      <strong>
+                        No matches found.
+                      </strong>
+                    </p>
+                    <p>
+                      Ensure you have entered the correct, full legal name of the organization before entering the phone
+                      number and mailing address.
+                    </p>
+                  </v-col>
+                </v-row>
+              </v-list-item-subtitle>
+            </v-list-item-content>
+          </v-list-item>
+        </v-list>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { SearchResponseI } from '@/interfaces' // eslint-disable-line no-unused-vars
+import { useSearch } from '@/composables/useSearch'
+import { BusinessTypes } from '@/enums/business-types'
+
+export default defineComponent({
+  name: 'BusinessSearchAutocomplete',
+  props: {
+    setAutoCompleteIsActive: {
+      type: Boolean
+    },
+    searchValue: {
+      type: String,
+      default: ''
+    },
+    showDropdown: {
+      type: Boolean
+    }
+  },
+  setup (props, { emit }) {
+    const { searchBusiness } = useSearch()
+
+    const localState = reactive({
+      autoCompleteIsActive: props.setAutoCompleteIsActive,
+      autoCompleteResults: [],
+      autoCompleteSelected: null,
+      showAutoComplete: computed((): boolean => props.searchValue.length >= 3),
+      searching: false,
+      isSearchResultSelected: false
+    })
+
+    const updateAutoCompleteResults = async (searchValue: string) => {
+      localState.searching = true
+      const response: SearchResponseI = await searchBusiness(searchValue)
+      // check if results are still relevant before updating list
+      if (searchValue === props.searchValue && response?.searchResults.results) {
+        localState.autoCompleteResults = response?.searchResults.results
+      }
+      localState.searching = false
+    }
+
+    const isSPGP = (businessType: string): boolean => {
+      return businessType === (BusinessTypes.GENERAL_PARTNERSHIP || BusinessTypes.SOLE_PROPRIETOR)
+    }
```

Comment:
This might actually clarify my comments above, are these types not supported via the look up? 
If so, than ignore my comments from above :)

Reply:
We need to check for SP or GP types and disable them in the list (and show tooltip).